ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/tables/agents.md

```diff
@@ -0,0 +1,24 @@
+<!--
```

Comment:
Need to also add this to `ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/input-variables.md`

Reply:
[9065c77](https://github.com/aristanetworks/ansible-avd/pull/3282/commits/9065c77cf24a353da2e1bd0c65dab4277f1b226a)